### PR TITLE
Access to model's last inserted ID

### PR DESCRIPTION
**Pull request by nowackipawel**
**Description**
Allows to get insertID after insert($insert_data, false) which returns CodeIgniter\Database\*\Result object instead of insertID.

### CODE REVIEW

**Comment by jim-parry** [comment]
There is an insertID() provided in Database\BaseConnection. Can it be used instead of a new method, with a different accessor style (getInsertID vs insertID)?

**Comment by nowackipawel** [comment]
Ofc there is insertID() method in BaseConnection but BaseConnection is not accessible from outside of the model  after insert is called with false as the 2nd parameter.

**Comment by jim-parry** [comment]
Fair enough. It just struck me as odd, but makes sense now.

File: system/Model.php [hunk 1]
```diff
@@ -551,6 +570,8 @@ public static function classToArray($data, string $dateFormat = 'datetime'): arr
 	public function insert($data = null, bool $returnID = true)
```
Comment (atefBB) [inline system/Model.php:570]: I think this feature isn't `well` documented! Need to be documented.

**Pull request merged**